File: Lab7Calificado/Lab7_2026_1/PokeSoftSolution/PokeSoftPersistance/TipoPokemonsDAO/DAO/TipoPokemonDAO.cs
using PokeSoftModel;
using PokeSoftPersistance.DAO;
using System;
using System.Collections.Generic;
using System.Text;

namespace PokeSoftPersistance.TipoPokemonsDAO.DAO
{
    public interface TipoPokemonDAO : IDAO<TipoPokemon>
    {
        TipoPokemon BuscarPorNombre(String nombre);
    }
}

File: lab11Calificado/Planchazo-Lab-11-Pokemon/PokemonsProg/PokemonesDAO/src/main/java/pe/edu/pucp/pokemones/dao/TipoPokemonDAO.java
package pe.edu.pucp.pokemones.dao;

import pe.edu.pucp.pokemones.dao.base.BaseDAO;
import pe.edu.pucp.pokemones.model.TipoPokemon;

public interface TipoPokemonDAO extends BaseDAO<TipoPokemon> {
    TipoPokemon buscarPorNombre(String nombre);

}
